app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -142,6 +142,23 @@ def billed?
           !!doc_auth_result
         end
 
+        def doc_auth_success?
+          transaction_status_passed? &&
+            true_id_product.present? &&
+            product_status_passed? &&
+            doc_auth_result_passed?
+        end
+
+        # @return [Boolean, nil]
+        # When selfie result is missing, return nil
+        # Otherwise:
+        #   return true if selfie check result == 'Pass'
+        #   return false
+        def selfie_success
```

Comment:
```suggestion
        def selfie_success?
```

Should we updated it to have a `?` at the end and be `selfie_success?` to match`doc_auth_success`? and other predicates?

Reply:
@zachmargolis , this returns nil|true|false, so strictly speaking it is not a boolean.

Reply:
having a result that distinguishes between `false` and `nil` is confusing, maybe should have switched to an enum-like thing with symbols?

Reply:
@dawei-nava @zachmargolis 
Hmm, are we thinking something like the below?

```
enum selfie_success: {
success: 0, #previously represented by true
fail: 1, # previously represented by false
no_selfie_processing: 3, #previously represented by nil
}	
```

(Caveat: above is a rough draft and the naming could definitely be improved)

I'm asking this for my own understanding: what are we hoping to avoid with this potential change?  Is there a concern that it's confusing to distinguish between `false` and `nil` because both are falsey?  (That's where my head went, but I want to make sure I'm not missing anything.)

Reply:
I was suggesting maybe the method jsut returns a symbol, ex `:success`, `:fail`, or `:no_selfie_processing`

Reply:
@zachmargolis Thanks for flagging this, we made a ticket. [LG-12159](https://cm-jira.usa.gov/browse/LG-12159)